code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -10335,10 +10335,39 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 		if(!locate(/obj/effect/decal/cleanable/purpledrank) in T)
 			new /obj/effect/decal/cleanable/purpledrank(T)
 
+/datum/reagent/punctualite
+	name = "Punctualite"
+	id = PUNCTUALITE
+	description = "Nicknamed mad chemist's alarm clock. Explodes on the hour when being actively metabolized. "
```

Comment:
Trailing space?

Reply:
Given that this doesn't actually metabolize, the phrasing is a little weird here. Why not "explodes on the hour while inside a living creature"?